### PR TITLE
feat: [#1401] load data from cloud (s3)

### LINKED ISSUE #1401: Load data from a cloud (S3 bucket)
Scope:
- [ ] Create a celery task that will load data from a link to S3 folder
- [ ] The task should return a python object (e.g. dict) where keys identify data type (preferably names that are used in our code - settings.py) and values enable quick and easy access to specific files from that directory. Do not load whole files there. Use generators or not - if you think that those values can be used multiple times. You can store full links to those files inside the directories. Dont story only names because then later on we will be forced to pass URL further in the code
- [ ] It should expect a specific dump structure - e.g. certain directory names. 
- [ ] It shouldn't expect a specific file extension - .json, .zip, ... are all fine 
- [ ] Do not require all directories to be present - e.g. alone folder called "dataset" is just fine - then just return dict with:
```
{
"dataset": <values>
}
```

### PR DESCRIPTION
**Pull request by wiktorflorian**
Closes #1401

### CODE REVIEW

File: transform/app/tasks/transform/dump/validate.py [hunk 1]
```diff
@@ -16,12 +24,31 @@ def validate_dump(prev_task_status: Optional[CeleryTaskStatus], s3_url: str) ->
...
+
+        # 3. Validate folder structure in the dump
+        validate_s3_directory_structure(s3_client, s3_url)
+
+        # 4. Validate file extensions
+        validate_files_extension(s3_client, s3_url)
+
+        logger.info("Dump validated successfully.")
+
+        return CeleryTaskStatus(status="validated").dict()
```
Comment (Michal-Kolomanski) [inline transform/app/tasks/transform/dump/validate.py:51]: Lets set the status as `success` or `failure` - please set `failure`  when it failed. Each task should assign the same statuses.
Lets make some python file called celery_tasks_statuses and there assign:
SUCCESS =  `success` ...
Later on import those and set inside celery tasks. If you can please refactor other celery tasks to use the same statuses

Comment (Michal-Kolomanski) [reply transform/app/tasks/transform/dump/validate.py:51]: or you can name it successful and unsuccessful - decide on your own :D

File: transform/app/tasks/transform/dump/load.py [hunk 2]
```diff
@@ -1,17 +1,31 @@
...
 
 
 @celery.task(name="load_dump")
-def load_dump(prev_task_status: Optional[CeleryTaskStatus], s3_url: str) -> dict:
+def load_dump(prev_task_status: Optional[CeleryTaskStatus], s3_url: str) -> Dict:
     """Task to load the dump"""
-    logger.info("Loading dump...")
-    # TODO
-    return CeleryTaskStatus(status="toBeImplemented").dict()
+    if prev_task_status and prev_task_status.get("status") == "validated":
```
Comment (Michal-Kolomanski) [inline transform/app/tasks/transform/dump/load.py:16]: "validated" will be refactored

File: transform/app/tasks/transform/dump/validate.py (hunk 1 above)
Comment (wiktorflorian) [reply transform/app/tasks/transform/dump/validate.py]: Added new `task_statuses.py` in `app.services.celery`, with 3 states, which are used when `CeleryTaskStatus` is returned

**Review by Michal-Kolomanski — APPROVED** [review]
Really good job :D please squash commits

**Pull request merged**